src/makeRobot.js

```diff
@@ -38,7 +38,83 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const evacuateCoordsX = 1400;
+  const evacuateCoordsY = 500;
+
+  const robot = {
+    name,
+    wheels,
+    version,
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      robot.coords.x = evacuateCoordsX;
+      robot.coords.y = evacuateCoordsY;
+    },
+
+    goForward(step) {
+      if (!step) {
+        robot.coords.y++;
+      }
```

Comment:
use deafult params, fix it everywhere
```suggestion
    goForward(step = 1) {
```